server/text_generation_server/models/bloom.py

```diff
@@ -51,7 +51,7 @@ def __init__(
             dtype = torch.float16 if dtype is None else dtype
         else:
             device = torch.device("cpu")
-            dtype = torch.float32
+            dtype = torch.bfloat16 if dtype == torch.bfloat16 else torch.float32
```

Comment:
```suggestion
            dtype = torch.float32 if dtype is None else dtype
```

Can you modify everywhere accordingly ?

Reply:
cpu doesnot support float16

Reply:
done